Source/Scene/Scene.js

```diff
@@ -463,6 +468,13 @@ define([
          */
         this.farToNearRatio = 1000.0;
 
+        /**
+         * The far-to-near ratio of the multi-frustum when using a logarithmic depth buffer. The default is 1e9.
+         * @type {Number}
+         * @default 1e9
+         */
+        this.logarithmicDepthFarToNearRatio = 1e9;
```

Comment:
Better document the interaction between `logarithmicDepthBuffer`, `farToNearRatio`, and `logarithmicDepthFarToNearRatio` and use `@see`/`@link` to link them together.  If I am following the boolean controls using one or the other ratios.